src/main/resources/check-partition-job.lua

```diff
@@ -0,0 +1,11 @@
+--lua
+local old_time=redis.call('get', KEYS[1])
```

Comment:
in place of this Lua script, can we use SETNX method of Redis: 
https://redis.io/commands/setnx

Reply:
"Design pattern: Locking with SETNX

Please note that:

The following pattern is discouraged in favor of the Redlock algorithm which is only a bit more complex to implement, but offers better guarantees and is fault tolerant..."

I think LUA script is safer and according to my testing so far no non-atomic instances occur with LUA script.

Reply:
can we have steps to test it in docs-internal repo ?

Reply:
That will be in the next PR. This PR provides JUnit only